target_iceberg/sinks.py

```diff
@@ -26,13 +24,14 @@ def __init__(self, target, schema, stream_name, key_properties) -> None:
             key_properties=key_properties,
         )
         table_name_prefix = f"{self.config.get('table_name_prefix')}_" if self.config.get("table_name_prefix") else ""
-        self.table_name = f"{self.config['db_name'] if self.config.get('prod') else 'scratch'}.{table_name_prefix}{self.__class__.to_snake_case(self.stream_name)}"
+        self.table_name = (f"{self.config['db_name'] if self.config.get('prod') else 'scratch'}."
+                           f"{table_name_prefix}{IcebergSink.to_snake_case(self.stream_name)}")
         self.flatten_max_level = self.config.get("max_flatten_level", 0)
         self.skip_add_synced_field = self.config.get("skip_add_synced_field", False)
+        self.overwrite_data = self.config.get("overwrite_data", False)
```

Comment:
It can be boolean as we can have incremental streams and full sync running with the same command. Let's make it a list of streams.
```suggestion
        self.overwrite_data = self.config.get("overwrite_data", '').split(',')
```